README.md

```diff
@@ -1,20 +1,40 @@
 # The Mapzen iOS SDK
 [![Circle CI](https://circleci.com/gh/mapzen/ios.svg?style=shield&circle-token=158f79f566b88fb913ad153ee8b00681112eb5a2)](https://circleci.com/gh/mapzen/ios)
 
+<p align=center>
+<img width="311" height="552" src="https://mapzen-assets.s3.amazonaws.com/images/ios-sdk-beta/sdk-beta-map.png">
+</p>
+
 The Mapzen iOS SDK is a thin wrapper that packages up everything you need to use Mapzen services in your iOS applications.
 
-We're currently under heavy development with this SDK. We welcome feature requests by posting an issue and labeling it appropriately as an ["Enhancement"](https://github.com/mapzen/ios/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement). Feel free to also reach out to us using any of the other channels we have available on https://mapzen.com/.
+We recently released our beta version of the SDK, version 0.2.0. The API at this point is fairly stable and we'd welcome feedback from the community on its usage. Feature requests are also welcome; label it appropriately as an ["Enhancement"](https://github.com/mapzen/ios/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) and be aware we probably won't be able to implement it until after launch at this point. Feel free to also reach out to us using any of the other channels we have available on https://mapzen.com/.
 
-## Usage
+# What's Included
 
-At Mapzen we're developing our various SDKs using Xcode 8 and Swift 2.3. However, minor changes are needed to run in Xcode 7 / Swift 2.2. If you need Swift 2.2 support, or you run into any other issues, please open an issue ticket on Github and we'll figure it out together! 
+Version 1 of our iOS SDK will be nearly featurity parity with our well established [Android SDK](https://github.com/mapzen/android). Major features include:
+- High performance and highly customizable mapping provided by [Tangram-es](https://github.com/tangrams/tangram-es).
```

Comment:
What about "map rendering" instead of "mapping"? We could also use "OpenGL ES" for the keywords, if this README is referenced from search somehow.